Functions/xASL_init_FileSystem.m

```diff
@@ -50,6 +50,9 @@
 xASL_adm_CreateDir(x.D.FLAIR_CheckDir);
 % xASL_adm_CreateDir(x.D.FLAIR_REGDIR);
 
+xASL_adm_CreateDir(x.D.T1c_CheckDir);
+xASL_adm_CreateDir(x.D.T2_CheckDir);
+
 xASL_adm_CreateDir(x.D.ASLCheckDir);
```

Comment:
Need to be careful not to always create these QC folders, 95% of the cases won't have a T2 or T1c.

Reply:
Shall we only create when needed? When this would go then?

Reply:
I see below. Done.